plugins/module_utils/dnac.py

```diff
@@ -193,7 +206,8 @@ def get_dnac_params(self, params):
                        "dnac_password": params.get("dnac_password"),
                        "dnac_verify": params.get("dnac_verify"),
                        "dnac_debug": params.get("dnac_debug"),
-                       "dnac_log": params.get("dnac_log")
+                       "dnac_log": params.get("dnac_log"),
+                       "dnac_log_level": params.get("dnac_log_level")
```

Comment:
Documentation of log_level is missing. Can you please check and update?

Reply:
It should be documented in the individual intent files, have sent instructions on webex. 

Changes required for each of intent module files and the playbook files
	1.	credentials.yml file - add the new parameter
		dnac_log_level: debug (Can be debug, info, warning, error, critical)
	2.	xyz_intent.py file

	•	In the Documentation and examples an additional parameter needs to be added below the dnac_log parameter
		dnac_log_level: “{{ dnac_log_level}}”
	•	in element_spec dictionary under the main() function add the parameter below dnac_log
		'dnac_log_level: {'type': ‘str’, 'default’:’debug’},
	•	In places where you are logging, you can modify the statements to the following based on the message
		If you dont modify the self.log statements immediately it will still work, the default is self.log(msg, level='info')
		self.log(msg, ‘debug’)  Lowest log level
		self.log(msg, ‘info’) All the significant information that needs to be log
		self.log(msg, ‘warning’) Potential Issues that may need attention
		self.log(msg, ‘error’) Errors that need immediate attention
		self.log(msg, ‘critical’) Highest log level

	3.	In the playbook xyz_intent.yml
		Add the new parameter below dnac_log
		dnac_log_level: “{{ dnac_log_level}}”
		OR
		dnac_log_level: 'debug'  (Can be debug, info, warning, error, critical)